sklearn/gaussian_process/tests/test_gpr.py

```diff
@@ -654,31 +676,81 @@ def test_gpr_predict_error():
         gpr.predict(X, return_cov=True, return_std=True)
 
 
-def test_y_std_with_multitarget_normalized():
-    """Check the proper normalization of `y_std` and `y_cov` in multi-target scene.
+@pytest.mark.parametrize("normalize_y", [True, False])
+@pytest.mark.parametrize("n_targets", [0, 1, 10])
```

Comment:
I don't like that `n_targets=0` means single target. I would instead isolate this case in a separate case or add the check in an existing test that is already checking it.

This way we can limit to 2D target with the case of having a single column or multicolumns.

Reply:
I see, it does look strange. I thought it fit well here since this test was checking all the output shapes and couldn't find another appropriate test.

How would you feel about setting `n_targets=None`?